### DISCUSSION TITLE
根据 音乐文件的 url 上传

### DISCUSSION THREAD

**Discussion by dudiao** [lulu-ls/cloud-uploader#54] [Announcements]
能否增加：根据 音乐文件的 url 进行上传

**Comment by lulu-ls** [comment]
url 指得是网络地址吗 ？

**Comment by dpeng** [comment]
> 能否增加：根据 音乐文件的 url 进行上传

这个有些难度了，网络存储有些还需要额外的认证，比如百度云、阿里云等。。。

开个脑洞。。。如果网易云音乐API里面如果支持网易云盘的音乐文件分享的话，作者可以考虑做个音乐库：-）

**Comment by lulu-ls** [comment]
如果由我来做个平台，让大家互相分享、上传、下载音乐，我不确定是不是有法律风险，如果没有，倒是挺好的 idea 🥇

**Comment by dpeng** [comment]
> 

网上有很多音乐分享平台，提供类似BBS的功能，自己不存储音乐资源，不确定是否有法律风险。小范围内不以盈利为目的的纯分享，应该不会有法律责任的吧，可以找个律师朋友问问

**Comment by lulu-ls** [comment]
如果是用户私下分享，应该是可以的，但是如果我提供下载平台，可能构成非法传播，但是我可以提供个交流平台，大家自己交换资源，我不保存和提供下载地址，但是这样不知道有没有意义

**Comment by dudiao** [comment]
是的，网络地址
